<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDDialog</class>
 <widget class="QDialog" name="SDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shape distribution</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/shapeDistribution</normaloff>:/icons/shapeDistribution</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="menuFrame">
     <property name="minimumSize">
      <size>
       <width>290</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="3" column="0">
       <spacer name="menuVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="optionsFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="0" rowspan="2" colspan="3">
          <widget class="QPushButton" name="calculateButton">
           <property name="text">
            <string>Calculate destriptor</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="binsSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1000</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save descriptor</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="optionsLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Destriptor options</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="samplesSpinBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1000</number>
           </property>
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>1048576</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="binsLabel">
           <property name="text">
            <string>Number of bins:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="samplesLabel">
           <property name="text">
            <string>Random samples:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="compareFrame">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="1">
          <widget class="QSpinBox" name="stepsSpinBox">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="descriptorComboBox">
           <item>
            <property name="text">
             <string>Distances to centroid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Distances between points</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle areas</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tetrahedra volumnes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Angles between vectors</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QPushButton" name="compareButton">
           <property name="text">
            <string>Compare destriptor with collection</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="endStepLabel">
           <property name="text">
            <string>Final step (dB)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="distanceMetricComboBox">
           <item>
            <property name="text">
             <string>Euclidian (pdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Euclidian (cdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>City block (pdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>City block (cdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chebychev (pdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chebychev (cdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimum coordinate (pdf)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimum coordinate (cdf)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="descriptorLabel">
           <property name="text">
            <string>Descriptor:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="distanceMetricLabel">
           <property name="text">
            <string>Distance metric:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="compareLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Model comparison</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="iniStepLabel">
           <property name="text">
            <string>Initial step (dB)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="iniStepSpinBox">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>-10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="stepsLabel">
           <property name="text">
            <string>Number of steps:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="endStepSpinBox">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="imgFrame">
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QFrame" name="opDistanceFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="qt_tools::plots::PlotWidget" name="opDistanceView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="tpDistanceFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="qt_tools::plots::PlotWidget" name="tpDistanceView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="tpAreaFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="qt_tools::plots::PlotWidget" name="tpAreaView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="fpVolumeFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="qt_tools::plots::PlotWidget" name="fpVolumeView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="tvAngleFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="qt_tools::plots::PlotWidget" name="tvAngleView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="imgHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qt_tools::plots::PlotWidget</class>
   <extends>QWidget</extends>
   <header>qt_tools/plots/PlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
